kano_feedback/DataSender.py

```diff
@@ -438,11 +438,17 @@ def try_connect():
     return is_internet()
 
 
-def send_question_response(question_id, answer, interactive=True):
+def send_question_response(answers, interactive=True, tags=['os', 'feedback-widget'],
```

Comment:
Maybe giving the answers as a list of tuples would be better? You could easily convert to this form from a list of `:` separated strings with

```python
[ answer.split(':') for answer in answers ]
```

But this way it stops us merging two responses only for them to be split up again. Also, `:` as a separator might be a bit of a trouble because the response might contain it.